test/torchaudio_unittest/transforms/autograd_test_impl.py

```diff
@@ -0,0 +1,45 @@
+from parameterized import parameterized
+from torch.autograd import gradcheck, gradgradcheck
+import torchaudio.transforms as T
+
+from torchaudio_unittest.common_utils import (
+    TestBaseMixin,
+    get_whitenoise,
+)
+
+
+class AutogradTestCase(TestBaseMixin):
+    def assert_grad(self, transform, *inputs, eps=1e-06, atol=1e-05, rtol=0.001):
+        transform = transform.to(self.device, self.dtype)
+
+        inputs_ = []
+        for i in inputs:
+            i.requires_grad = True
+            inputs_.append(i.to(dtype=self.dtype, device=self.device))
```

Comment:
nit you might want to assert that the dtype is float64 here because otherwise this test, with default eps/tolerance will fail.